app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int, **kwargs) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
```

Comment:
we accept list of dicts (`list[dict]`) and return list of Persons (`list[Person]`) 

Also to use a class in annotation import `from __future__ import annotations`